openapi/sled-agent.json

```
@@ -1040,7 +1040,7 @@
         "title": "An IPv6 subnet",
         "description": "An IPv6 subnet, including prefix and subnet mask",
         "type": "string",
-        "pattern": "^([fF][dD])[0-9a-fA-F]{2}:(([0-9a-fA-F]{1,4}:){6}[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,6}:)\\/(6[4-9]|[7-9][0-9]|1[0-1][0-9]|12[0-6])$"
+        "pattern": "^([fF][dD])[0-9a-fA-F]{2}:(([0-9a-fA-F]{1,4}:){6}[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,6}:)\\/(6[4-9]|[7-9][0-9]|1[0-1][0-9]|12[0-6]|4[0-9])$"
```

After talking with @bnaecker today, I think we're going to need to check the prefix anyways in higher level software. My suggestion is that we allow a prefix range from 1-128 inclusive to just avoid dealing with this again in the future.

This regex restricts us to `fd00/8`. In the future, I think we may want to relax this further to just rely on what's in `ipnetwork::Ipv6Network` as of v0.20.0.

For now here's what I suggest:

```suggestion
        "pattern": "^([fF][dD])[0-9a-fA-F]{2}:(([0-9a-fA-F]{1,4}:){6}[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,6}:)\\/([1-9]|[1-9][0-9]|1[0-1][0-9]|12[0-8])$"
```

---

Sounds good!